egui/src/data/input.rs

```diff
@@ -177,6 +177,8 @@ pub enum Event {
     /// The mouse or touch moved to a new place.
     PointerMoved(Pos2),
 
+    MouseDelta(Vec2),
```

Comment:
This need documentation!

For instance: should it always be emitted alongside `PointerMoved`? Is it emitted on touch screens?

Reply:
Well, the whole solution is workaround . It `needs` to be exact `one` event with position and delta both. But as we can see, winit has different event for window-mouse-position change, and DeviceEvent.

For now I use MouseDelta when it emmited, if not - there is fallback to position, so is would not affect touch screens.

I'll add this note is comment